server/src/main/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplateServicesImpl.java

```diff
@@ -106,32 +106,30 @@ public FeedbackTemplate getById(UUID id) {
         return feedbackTemplate.get();
     }
 
-
     @Override
     public List<FeedbackTemplate> findByFields(@Nullable UUID creatorId, @Nullable String title) {
         if (!getIsPermitted()) {
             throw new PermissionException("You are not authorized to do this operation");
         }
 
         List<FeedbackTemplate> templateList = new ArrayList<>();
-        // Filters only active templates by default
+
         if (title != null) {
```

Comment:
This logic can be simplified if you write custom query in repository method. then you don't need to use retainall or addall at all and query will do all the work. Take a look at the query in OpportunitiesRepository.java. We can discuss if you have questions.